mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -1,16 +1,139 @@
 CLASS net/minecraft/class_1 net/minecraft/entity/Entity
+	FIELD field_1013 onGround Z
+	FIELD field_1014 horizontalCollision Z
+	FIELD field_1015 removed Z
+	FIELD field_1016 yOffset F
+	FIELD field_1017 pitch F
+	FIELD field_1018 prevDistanceTraveled F
+	FIELD field_1019 distanceTraveled F
+	FIELD field_1021 fallDistance F
+	FIELD field_1022 nextFlySoundDistance D
+	FIELD field_1023 lastRenderX D
+	FIELD field_1024 lastRenderY D
+	FIELD field_1025 stepHeight F
+	FIELD field_1026 age I
+	FIELD field_1028 yaw F
+	FIELD field_1029 nextStepSoundDistance I
+	FIELD field_1030 lastRenderZ F
 	FIELD field_1031 world Lnet/minecraft/class_14;
+	FIELD field_1032 prevX D
+	FIELD field_1033 prevY D
+	FIELD field_1034 prevZ D
 	FIELD field_1035 x D
 	FIELD field_1036 y D
 	FIELD field_1037 z D
-	FIELD field_1261 ticksOnFire I
-	FIELD field_1691 skinUrl Ljava/lang/String;
+	FIELD field_1038 velocityX D
+	FIELD field_1039 velocityY D
+	FIELD field_1040 velocityZ D
+	FIELD field_1041 yaw F
+	FIELD field_1042 pitch F
+	FIELD field_1043 prevYaw F
+	FIELD field_1044 prevPitch F
+	FIELD field_1168 random Ljava/util/Random;
+	FIELD field_1220 noClip Z
+	FIELD field_1221 pushSpeedReduction F
+	FIELD field_1260 fireResistance I
+	FIELD field_1261 fireTicks I
+	FIELD field_1345 maxAir I
+	FIELD field_1346 timeUntilRegen I
+	FIELD field_1347 air I
+	FIELD field_1348 touchingWater Z
+	FIELD field_1512 firstUpdate Z
+	FIELD field_1691 skinURL Ljava/lang/String;
```

Comment:
Would prefer Url but is just kinda personal convention; and I don't think we ever decided on a convention for here.